### PR TITLE
fix: change breakpoints on new page wrapper

### PR DESCRIPTION
**Pull request by storyworks**
# Description

### <samp>🤖 Generated by Copilot at 84bd7d0</samp>

This pull request adds a new navigation drawer component to the journeys admin app, which allows users to navigate between different pages and features. It also updates the media query breakpoints for the drawer layout and other components to improve the consistency and responsiveness of the app. Additionally, it adds test files, story files, and helper functions for the new components and features. The files affected by this pull request are `HelpScoutBeacon.tsx`, `PageWrapper.tsx`, `SidePanel.tsx`, `AddJourneyButton.tsx`, `AppHeader.tsx`, and all the files under the `NavigationDrawer` and `getJourneyTooltip` directories.

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/32318254/todos/6051671986)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [x] Desktop layout switches to mobile at this point 
image

- [ ] Navigation Bars, fab, helpscout beacon still display in correct place
- [ ] Fix only occurs on landing page

# Walkthrough

### <samp>🤖 Generated by Copilot at 84bd7d0</samp>

*  Change the media query breakpoints for the drawer layout from `sm` to `md` to improve the responsiveness and consistency of the app ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-d5f6c8eed9cbc9c4a46c3d1d0fd94808bf098624879fe45d7b4c6cf25243a40dL38-R38), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-d5f6c8eed9cbc9c4a46c3d1d0fd94808bf098624879fe45d7b4c6cf25243a40dL50-R50), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-c6725274cd1372247002b3d87916587bddeeb5a2faf7737aa981f9176a369d75L210-R210), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-43f2ca7c715a6450ef736ef3e32753e9eb3105ae67f31553f8782b56c98a8d1fL20-R25), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-5965e5f2f61464ac798b2d978eb50c82874afd08713ad90b1fbd61c962f04237L54-R68), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-5965e5f2f61464ac798b2d978eb50c82874afd08713ad90b1fbd61c962f04237L79-R79), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-3b413ddbefdf49daf5a4fcc8033f9e454a35a5dea6c8f98c746c1b3237d784afL86-R86), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-3b413ddbefdf49daf5a4fcc8033f9e454a35a5dea6c8f98c746c1b3237d784afL107-R107))
* Create a new navigation drawer component that displays a sidebar menu with the app logo, title, user profile, and navigation items ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-24151e0fb3ef8e0b5c22ecf5dc45ac477594c019f117933a4cfcc48ac7caa4fcR1-R243), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-fab74a232c5e444c98721a8810fb54ba8c76b62c5a5a51a3df9b9db480a7dc44R1), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-33b271b15c2ed76efd91c6c9789b4b50c97f191ce9e7e581213bf156918cf7ebR1), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-7dc7cc92319044e9ac702a435deba50446be70442bb8a84faa13ebff4d479808R1))
  * Add test cases for the navigation drawer component and its subcomponents using `NavigationDrawer.spec.tsx`, `NavigationListItem.spec.tsx`, and `UserMenu.spec.tsx` ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-0b66a55c4389516b7cf55773a557d9b57bfe721b4f217c94655b920ef4d55b09R1-R295), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-91ef520da0a15f4d1d71a8a9b6806d438e646415211ee149d6c6becfcd44195fR1-R96), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-8b2a53761202e356700c80e94ee8dfa95280b211f2e00e912df59f5a02d1b906R1-R70))
  * Add stories for the navigation drawer component and its subcomponents using `NavigationDrawer.stories.tsx`, `NavigationListItem.stories.tsx`, and `UserMenu.stories.tsx` ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-27ea62121bd0529ee73a29273085be0849f468ea2c26f6f39a886ea1c6387126R1-R117), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-bea4cb9afd6b7d7ae49ae6cd089c6cd987abf307c8208c572b3ea04bfd824b39R1-R41), [link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-1c803304040dc4ca9a4f82bd27d9017e93df50f6d49ca7a571836abad932fa7eR1-R48))
  * Update the import path for the navigation drawer component in `PageWrapper.tsx` ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-5965e5f2f61464ac798b2d978eb50c82874afd08713ad90b1fbd61c962f04237L7-R8))
* Create a new navigation list item component that displays a menu item with an icon, a label, a link, a tooltip, and a badge ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-9d52cfb295ad2b3346363ad70f5bf60753ef1c43b2360436bcdea8e87d364c46R1-R82))
* Create a new user menu component that displays a dropdown menu with the user profile information and a log out option ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-87bc7b267d4654f1b6267a7210d57f45409d5ec83ccdb98fa25a3d111e33839aR1-R77))
* Create a new getJourneyTooltip function that returns a tooltip text for the discover menu item based on the user journeys data ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-6af98aa74af169001d2f2ad70d52fa3a6cc80047c856d988150bc9bc7bf17ef1R1-R35))
  * Add test cases for the getJourneyTooltip function using `getJourneyTooltip.spec.tsx` ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-0d5c3d3ae66d9e15e648e50bb799b8a9c10d208687da0fb476c963c72e027477R1-R91))
  * Export the getJourneyTooltip function from its index file ([link](https://github.com/JesusFilm/core/pull/1482/files?diff=unified&w=0#diff-ce23092decc21a469e214910896204a3f1ecf69b8e950f982072da57c91c89d3R1))

### CODE REVIEW

File: apps/journeys-admin/src/components/NewPageWrapper/utils/getJourneyTooltip/getJourneyTooltip.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { GetJourneys_journeys as Journey } from '../../../../../__generated__/GetJourneys'
+import { UserJourneyRole } from '../../../../../__generated__/globalTypes'
+
+export function getJourneyTooltip(
```
Comment (storyworks) [inline apps/journeys-admin/src/components/NewPageWrapper/utils/getJourneyTooltip/getJourneyTooltip.ts:4]: Copy pasted from `PageWrapper` so we can delete that all in one go when redundant.

File: apps/journeys-admin/src/components/NewPageWrapper/NavigationDrawer/NavigationDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+      id
+      firstName
+      lastName
+      email
+      imageUrl
+    }
+  }
+`
+
+const StyledNavigationDrawer = styled(Drawer)(({ theme, open }) => ({
```
Comment (storyworks) [inline apps/journeys-admin/src/components/NewPageWrapper/NavigationDrawer/NavigationDrawer.tsx:58]: Everything in the `NavigationDrawer` folder has been copy pasted into "NewPageWrapper" folder so we can easily refactor when removing the old PageWrapper.

This Nav drawer also opens on `mdUp` whereas the original opens on `smUp`

File: apps/journeys-admin/src/components/NewPageWrapper/NavigationDrawer/NavigationDrawer.tsx [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+export function NavigationDrawer({
+  open,
+  onClose,
+  authUser,
+  router
+}: NavigationDrawerProps): ReactElement {
+  const activeJourneys = useActiveJourneys()
+  const journeys = activeJourneys?.data?.journeys
+  const { t } = useTranslation('apps-journeys-admin')
+  const mdUp = useMediaQuery((theme: Theme) => theme.breakpoints.up('md'))
```
Comment (storyworks) [inline apps/journeys-admin/src/components/NewPageWrapper/NavigationDrawer/NavigationDrawer.tsx:109]: Everything in the `NavigationDrawer` folder has been copy pasted into "NewPageWrapper" folder so we can easily refactor when removing the old PageWrapper.

This Nav drawer also opens on `mdUp` whereas the original opens on smUp

**Comment by GeronimoJohn** [comment]
Reports as current page but Nav Drawer not reflecting that

Screenshot 2023-05-12 at 11 38 20 AM

**Review by GeronimoJohn — APPROVED** [review]

**Review by GeronimoJohn — DISMISSED** [review]

**Review by GeronimoJohn — APPROVED** [review]

**Pull request merged**